devices/src/lib.rs

```diff
@@ -41,6 +41,14 @@ pub type DeviceEventT = u16;
 pub enum EpollHandlerPayload {
     /// DrivePayload(disk_image)
     DrivePayload(File),
+    /// Used to mutate current RateLimiter settings. The buckets are rx_bytes, rx_ops,
+    /// tx_bytes, and tx_ops, respectively.
+    PatchRateLimiters(
```

Comment:
For consistency reasons we should name this RateLimitersPayload or something of that sort.

Reply:
Done.